service/version.go

```diff
@@ -21,12 +21,18 @@ func (uis *UIServer) versionPage(w http.ResponseWriter, r *http.Request) {
 	projCtx := MustHaveProjectContext(r)
 	project, err := projCtx.GetProject()
 
-	if err != nil {
-		http.Error(w, err.Error(), http.StatusInternalServerError)
+	grip.DebugWhen(err != nil, message.Fields{
```

Comment:
Should we also log when project or projectCtx is nil as well? Since those were the conditions of the original error